algorithms/unit_tests/TestSort.hpp

```diff
@@ -432,12 +432,20 @@ void test_issue_4978_sort() {
 template <class ExecutionSpace, typename KeyType>
 void test_sort(unsigned int N) {
   test_1D_sort<ExecutionSpace, KeyType>(N);
-  test_3D_sort<ExecutionSpace, KeyType>(N);
+#if defined(KOKKOS_ENABLE_CUDA) && \
+    defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC
```

Comment:
What's the problem? Do you just get wrong results?

Reply:
Wrong result

Reply:
And I see know the code is wrong I only meant to guard the runtime if

Reply:
> And I see know the code is wrong I only meant to guard the runtime if

Fixed